### PR TITLE
Modifying alerting logic based on alert classifications

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/commons#765

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @nnDarshan @r0h4n please review

**Comment by GowthamShanmugam** [comment]
Tendrl/gluster-integration#449

**Comment by GowthamShanmugam** [comment]
verified

File: tendrl/commons/objects/alert/__init__.py [hunk 1]
```diff
@@ -111,17 +111,17 @@ def is_same(self, alert1, alert2):
...
+        if "node" in alert1.classification and alert1.node_id:
+            if "node" in alert2.classification and alert2.node_id:
                 if alert1.node_id != alert2.node_id:
                     return False
             else:
                 return False
-        elif alert2.node_id:
+        elif "node" in alert2.classification and alert2.node_id:
             return False
         if alert1.alert_type != alert2.alert_type:
```
Comment (shtripat) [inline tendrl/commons/objects/alert/__init__.py]: Add a condition as below

```
if alert1.alert_type != alert2.alert_type or set(alert1.classification) != set(alert2.classification):
    return False
```

**Pull request merged**